### ISSUE TITLE
Warning: Could not load network parameter file!

### ISSUE THREAD

**Issue by johannesbjork** [YDaiLab/MiMeNet#3]
Running the provided examples, I was getting `Warning: Could not load network parameter file!`.
It states in the documentation that the `net_params` should be provided as a `.JSON` file, but the examples are given as `.txt` (although in JSON format). Re-saving it as a JSON file did not solve the issue. However, looking into the MiMeNet source code, I saw the hyper parameter for learning rate is denoted `learning_rate` in the hyper parameter file but `lr` in the source code, so fixing that resolved the issue.

```
{
	"num_layer": 1,
	"layer_nodes": 128.0, 
	"l1": 1e-03,
	"l2": 1e-03,
	"dropout": 0.25,
	"lr": 0.0001
}
```
